apps/meteor/client/views/room/Sidepanel/SidepanelItem/SidePanelParentRoom.tsx

```diff
@@ -0,0 +1,14 @@
+import type { SubscriptionWithRoom } from '@rocket.chat/ui-contexts';
+
+import SidePanelParentDiscussion from './SidePanelParentDiscussion';
+import SidePanelParentTeam from './SidePanelParentTeam';
+
+const SidePanelParentRoom = ({ room }: { room: SubscriptionWithRoom }) => {
+	if (room.prid) {
+		return <SidePanelParentDiscussion room={room} />;
+	}
```

Comment:
Like here for example we already check if `prid` is present in the room. If we make a type assertion function like "isDiscussionSubscription" or something similar, we can specify the type in the component a little more since we're guaranteeing this property exists before it even get's there.